script/install

```diff
@@ -20,10 +20,8 @@ warn() {
   printf "${tty_red}Warning${tty_reset}: %s\n" "$(chomp "$1")"
 }
 
-LATEST_VERSION=$(curl --silent "https://api.github.com/repos/tuist/tuist/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-
 ohai "Downloading tuistenv..."
-curl -LSs --output /tmp/tuistenv.zip https://github.com/tuist/tuist/releases/download/${LATEST_VERSION}/tuistenv.zip
+curl -LSs --output /tmp/tuistenv.zip https://storage.googleapis.com/tuist-releases/latest/tuistenv.zip
```

Comment:
I remembre Pedro's PRs also removed some uploading to GCS, shouldn't we restore it as well?